lib/jekyll/theme_builder.rb

```diff
@@ -56,12 +56,27 @@ def write_file(filename, contents)
 
   def create_directories
     mkdir_p(SCAFFOLD_DIRECTORIES)
+    mkdir_p("_sass/#{name}")
   end
 
   def create_starter_files
     %w(page post default).each do |layout|
       write_file("_layouts/#{layout}.html", template("_layouts/#{layout}.html"))
     end
+
+    %w(footer head header).each do |incl|
+      write_file("_includes/#{incl}.html", template("_includes/#{incl}.html"))
+    end
+
+    %w(github twitter).each do |icon|
+      write_file("_includes/icon-#{icon}.svg", template("_includes/icon-#{icon}.svg"))
+      write_file("_includes/icon-#{icon}.html", template("_includes/icon-#{icon}.html"))
+    end
```

Comment:
I added this because, I'm copying all required files from `minima` without any change.
Since, minima's footer include references these assets, I added them as well.